aws-iam-role/main.tf

```diff
@@ -1,4 +1,8 @@
 locals {
+  // Slight hack so Terraform can get the size statically during the plan. 
+  // Simply passing the list to `for_each` throws an Invalid for_each argument
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Single line comments should begin with #